16_FirstContribution/StonePaperScissors

```diff
@@ -0,0 +1,59 @@
+import java.util.Scanner;
+import java.util.Random;
+
+class spsgame {
```

Comment:
The class name should be same as the file name.

Also the file name should end with `.java` extension

Reply:
It's not compulsory unless the class is public